test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeSystemArraycopyTests.java

```diff
@@ -837,12 +769,22 @@ static public void testSystemArrayCopy26() throws Throwable {
 	static public void testSystemArrayCopy27() throws Throwable {
 		initArrays();
 		testVTVT(vtArraySrc, nullRestrictedVtArrayDst);
+		checkResults(vtArraySrc, nullRestrictedVtArrayDst);
+
+		initArrays();
+		testVTVT(vtArraySrc, nullRestrictedVtArrayDst);
+		checkResults(vtArraySrc, nullRestrictedVtArrayDst);
```

Comment:
Could you use Henry's suggestion, https://github.com/eclipse-openj9/openj9/pull/20250#discussion_r1786500488 instead of copying/paste the code twice like I initially suggested?

Reply:
Same comment applies to testSystemArrayCopy28 and testSystemArrayCopy29 as well